masonry/src/widgets/text_area.rs

```diff
@@ -26,6 +26,11 @@ use crate::theme::default_text_styles;
 use crate::util::debug_panic;
 use crate::{TextAlign, theme};
 
+/// Convert `parley::BoundingBox` into `vello::kurbo::Rect`
+fn convert_rect(rect: parley::BoundingBox) -> vello::kurbo::Rect {
+    vello::kurbo::Rect::new(rect.x0, rect.y0, rect.x1, rect.y1)
+}
```

Comment:
What do you think about copying this function into the Parley changelog as well?